spec/support/fakes/bundler/audit/task.rb

```diff
@@ -0,0 +1,11 @@
+require "pathname"
```

Comment:
As you will see down below, we are sticking `spec/support/fakes/bundler` to `LOAD_PATH` like this:

```ruby
expect(`RAILS_ENV=development rake -I#{fakes_path} -rfake_rails -T`)
```

The bundler audit rake task that we're testing runs the following code:

```ruby
if Rails.env.development? || Rails.env.test?
  require "bundler/audit/task"
  Bundler::Audit::Task.new
end
```

That `require` will actually hit this Ruby file, and on `Bundler::Audit::Task.new` it will output `Fake task loaded`, which is a side-effect we can use in our assertion.